docs/developer-guide/plugins.md

```diff
@@ -362,4 +362,32 @@ To make a single module provide multiple rules, export an array of plugin object
 
 ## Sharing plugins and plugin packs
 
-Use the `stylelint-plugin` keyword within your `package.json`.
+Use the `stylelint-plugin` keyword within your `package.json`. For example:
+
+```json
+{
+  "keywords": ["stylelint", "stylelint-plugin"]
+}
+```
+
+## Sharing configurations
+
+You may want to create a shareable configuration for your plugins to provide a default ruleset. For example:
+
+```json
+{
+  "plugins": ["stylelint-plugin-foo"],
+  "rules": {
+    "foo/rule-1": true,
```

Comment:
question
minor

Should be `stylelint-plugin-foo/rule-x`?

Reply:
Actually, we don't need to keep consistency between a package name and a rule namespace. So, I think it's good enough as it is.

E.g.

https://github.com/stylelint/stylelint/blob/67c21375e12f695a3929bde27281d772dcdeaf93/docs/developer-guide/plugins.md?plain=1#L27-L35